### DISCUSSION TITLE
Display installed versions and most recently released versions of ComfyUI components

### DISCUSSION THREAD

**Discussion by dryodryo** [Comfy-Org/ComfyUI_frontend#1105] [Ideas]
It would be super helpful if the About page listed the installed and most recently released versions of ComfyUI components. It only displays the installed versions. From within ComfyUI, there's no way of knowing if the components are up to date or not. This is especially important to know in ComfyUI because there are so many dependencies, therefore many things that can go wrong.

Of course, users can check the most recently released versions using the handy links to the various GitHub pages. But it would save users a lot of time if the About page could just pull that info and display it for us.

Thanks!

**Comment by christian-byrne** [comment]
How would you implement it? I think ComfyUI tries to avoid making any external network connections.

**Comment by dryodryo** [comment]
If there are no external network connections permitted, then I guess it's not possible. But maybe connections to specific domains or servers could be permitted?

**Reply by huchenlei** [reply]
We can do an explicitly triggered connection, i.e. the connection won't be automatically triggered without user consent. When user clicks the "Check version" button, we make the request to fetch version info.

**Reply by dryodryo** [reply]
That sounds perfect, thanks!
